CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# How to contribute
+
+If you haven't already, come find us in IRC ([##zuzakistan-lab](irc://chat.freenode.net/zuzakistan-lab) on freenode).
+
+## Suggesting a feature
+
+[Open an issue](https://github.com/zuzakistan/civilservant/issues) with your suggestion.
+
+## Finding something to work on
+
+Any of our issues can be picked up by anyone.
+We have marked a few issues as [especially good for new contributors](https://github.com/zuzakistan/civilservant/labels/good%20first%20issue).
+
+## Contributing code
+
+Fork, then clone the repo:
+
+    $ git clone git@github.com:your-username/civilservant.git
+
+Install dependencies:
+
+    $ npm install
+
+Make sure the tests pass:
+
+    $ npm test
+
+Make your change. We use [StandardJS](https://standardjs.com/rules.html) as our coding style.
+If you can, add tests for your change. Then make sure the tests pass:
+
+    $ npm test
```

Comment:
Does this need `npm test` twice?

Reply:
No, but I guess it's good practice to ensure the tests actually work before relying on them